deployment/terraform/strings.go

```diff
@@ -17,6 +17,7 @@ WorkingDirectory=/opt/mattermost
 User=ubuntu
 Group=ubuntu
 LimitNOFILE=49152
+Environment=MM_FEATUREFLAGS_INSIGHTSENABLED=true
```

Comment:
It would be nice if we could find a way to inject this through config instead of hard-coding it there.

Reply:
https://github.com/mattermost/mattermost-server/pull/20509 this is not needed anymore. So removed this line.